setup.py

```diff
@@ -1,55 +0,0 @@
-"""
-This is a file to describe the Python module distribution and
-helps with installation.
-
-More info on various arguments here:
-https://setuptools.readthedocs.io/en/latest/setuptools.html
-"""
-from setuptools import setup, find_packages
-from subprocess import check_output
-
-
-def get_version():
-    # https://github.com/uc-cdis/dictionaryutils/pull/37#discussion_r257898408
-    try:
-        tag = check_output(
-            ["git", "describe", "--tags", "--abbrev=0", "--match=[0-9]*"]
-        )
-        return tag.decode("utf-8").strip("\n")
-    except Exception:
-        raise RuntimeError(
-            "The version number cannot be extracted from git tag in this source "
-            "distribution; please either download the source from PyPI, or check out "
-            "from GitHub and make sure that the git CLI is available."
-        )
-
-
-def get_readme():
-    with open("README.md", "r") as f:
-        return f.read()
-
-
-setup(
-    name="gen3utils",
-    version=get_version(),
-    description="Gen3 Library Template",
-    long_description=get_readme(),
```

Comment:
will poetry include this by default in the pypi release? if not how can we add it?

Reply:
Yeah it will. With `readme = "README.md"` added to the `[tool.poetry]` section of `pyproject.toml`, Poetry will automatically set `long_description` to a string representing the contents of `README.md`.

Reply:
awesome!